src/tools/rados/rados.cc

```diff
@@ -176,6 +176,8 @@ void usage(ostream& out)
 "        select given pool by name\n"
 "   --target-pool=pool\n"
 "        select target pool by name\n"
+"   --pgid pg id\n"
+"        select given pg id\n"
```

Comment:
we are using both "PG" and "pg" interchangeably in the help message. could you take this opportunity to unify them? i'd suggest use "PG" in the user visible document and messages. as it's an abbreviation  of placement group. what do you think?